web-ng/src/app/components/layer-list/layer-list.component.scss

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+ @import '~src/custom-theme';
+
 .layer-list-title {
   padding: 20px;
 }
+
+.add-layer {
+  padding: 10px;
+}
+
+.add-layer-btn {
+  text-align: left;
+  width: 100%;
```

Comment:
Would it make sense to use Flex layouts here instead of 100% width and hard-coded padding? I did this recently for the sharing dialog:

https://github.com/google/ground-platform/blob/49fbb5076e64a7850d2e6f6094762b5f556b730d/web-ng/src/app/components/share-dialog/share-dialog.component.html#L21-L44

Could this also work here?